pom.xml

```diff
@@ -93,7 +93,7 @@
 
   <modules>
```

Comment:
Please **add** a maven profile in the main pom which will involve the format/format instead of format/format-java
This will be helpful to run in CI where we would also want to ensure the thrift files and the source code are in sync.